[2023-06-14T21:42:23.602+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: deploy_prediction_model.pyspark_train_classifier_model manual__2023-06-14T21:42:18.983310+00:00 [queued]>
[2023-06-14T21:42:23.608+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: deploy_prediction_model.pyspark_train_classifier_model manual__2023-06-14T21:42:18.983310+00:00 [queued]>
[2023-06-14T21:42:23.609+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 4
[2023-06-14T21:42:23.622+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): pyspark_train_classifier_model> on 2023-06-14 21:42:18.983310+00:00
[2023-06-14T21:42:23.628+0000] {standard_task_runner.py:57} INFO - Started process 687 to run task
[2023-06-14T21:42:23.643+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'deploy_prediction_model', 'pyspark_train_classifier_model', 'manual__2023-06-14T21:42:18.983310+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/deploy.py', '--cfg-path', '/tmp/tmppy5dpi6v']
[2023-06-14T21:42:23.647+0000] {standard_task_runner.py:85} INFO - Job 25: Subtask pyspark_train_classifier_model
[2023-06-14T21:42:23.725+0000] {task_command.py:410} INFO - Running <TaskInstance: deploy_prediction_model.pyspark_train_classifier_model manual__2023-06-14T21:42:18.983310+00:00 [running]> on host 6426fae5ea9e
[2023-06-14T21:42:23.827+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='deploy_prediction_model' AIRFLOW_CTX_TASK_ID='pyspark_train_classifier_model' AIRFLOW_CTX_EXECUTION_DATE='2023-06-14T21:42:18.983310+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-14T21:42:18.983310+00:00'
[2023-06-14T21:42:23.829+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-14T21:42:23.830+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '\nspark-submit --class org.fiware.cosmos.orion.spark.connector.prediction.PredictionJob    --conf spark.driver.extraJavaOptions=-Dlog4jspark.root.logger=WARN,console   --master spark://spark-master:7077   //prediction-job/target/orion.spark.connector.prediction-1.0.1.jar   /']
[2023-06-14T21:42:23.845+0000] {subprocess.py:86} INFO - Output:
[2023-06-14T21:42:23.847+0000] {subprocess.py:93} INFO - /bin/bash: line 2: spark-submit: command not found
[2023-06-14T21:42:23.848+0000] {subprocess.py:97} INFO - Command exited with return code 127
[2023-06-14T21:42:23.869+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 211, in execute
    f"Bash command failed. The command returned a non-zero exit code {result.exit_code}."
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 127.
[2023-06-14T21:42:23.872+0000] {taskinstance.py:1350} INFO - Marking task as UP_FOR_RETRY. dag_id=deploy_prediction_model, task_id=pyspark_train_classifier_model, execution_date=20230614T214218, start_date=20230614T214223, end_date=20230614T214223
[2023-06-14T21:42:23.883+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 25 for task pyspark_train_classifier_model (Bash command failed. The command returned a non-zero exit code 127.; 687)
[2023-06-14T21:42:23.930+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-06-14T21:42:23.950+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
